webpack.prod.js

```diff
@@ -0,0 +1,8 @@
+const { merge } = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+const prodConfig = {
+    mode: "production",
+    devtool: "source-map"
```

Comment:
only if you want sourcemaps in production

Reply:
d'oh! That isn't what I intended - good catch.